lib/bolt/cli.rb

```diff
@@ -111,7 +111,10 @@ def parse
 
       validate(options)
 
-      @config = if options[:configfile]
+      @config = if !Bolt::Util.windows? && ENV['BOLT_PROJECT']
+                  project = Bolt::Project.create_project(ENV['BOLT_PROJECT'], 'environment')
```

Comment:
We should call `expand_path` to support `~` and relative paths

Reply:
`Bolt::Project.create_project` already calls `Pathname.new(path).expand_path`.